usr/harvey/lib/profile

```diff
@@ -1,5 +1,6 @@
 bind -a $home/bin/rc /bin
 bind -a $home/bin/$cputype /bin
+bind -a /apex/amd64/bin /bin
```

Comment:
if this bind fails will the script break and stop here?

Reply:
No, I've tried it; an error is printed, but the script continues